scripts/release.sh

```diff
@@ -3,31 +3,32 @@
 set -e
 trap 'echo "******* FAILED *******" 1>&2' ERR
 
-USER=$(whoami)
-INSTALL_DIR=/home/$USER/aebot/build
```

Comment:
We should care if you delete this, as some nodes on the mainnet may broke since "/home/$USER "is not the same as "~"